webpack.prod.config.js

```diff
@@ -13,6 +15,17 @@ function resolvePrivateEnvConfig(filePath) {
 }
 resolvePrivateEnvConfig('.env.private');
 
+const plugins = []
+// PurgeCSS
+if (process.env.USE_PURGECSS) {
```

Comment:
If re-using the same `USE_PURGECSS` env var, the default production Webpack config from `@edx/frontend-build` will still include `@fullhuman/postcss-purgecss` as a PostCSS plugin, regardless of whether you include `PurgeCSSPlugin`.

E.g., if you console.log the `config`:

```
console.log(config.module.rules[2].use[2].options.postcssOptions);
```

```
{
  plugins: [
    {
      postcssPlugin: 'autoprefixer',
      prepare: [Function: prepare],
      info: [Function: info],
      options: {},
      browsers: undefined
    },
    { postcssPlugin: 'postcss-rtlcss', Once: [Function: Once] },
    Processor { version: '8.4.26', plugins: [Array] },
    {
      postcssPlugin: 'postcss-custom-media',
      prepare: [Function: prepare]
    },
    {
      postcssPlugin: 'postcss-purgecss',
      OnceExit: [Function: OnceExit]
    }
  ]
}
```

https://github.com/openedx/frontend-build/blob/35415fe423a6f1103eacf1b761c003a670537726/config/webpack.prod.config.js#L32-L36